build.xml

```diff
@@ -132,17 +132,76 @@
 	</target>
 
     <target name="prodVersion">
-            <!-- Full version examples: 1.0.0-SNAPSHOT-0-HASH, 1.0.0-0-HASH, 1.0.0-1-HASH, etc -->
-            <exec outputproperty="fullVersion" dir="." executable="sh">
-			<arg value="-c" />
-                <arg value="git describe --long | tr -d v" />
-		</exec>
-            <!-- Version examples: 1.0.0-SNAPSHOT, 1.0.0, etc -->
-            <exec outputproperty="releaseVersion" dir="." executable="sh">
-                <arg value="-c" />
-                <arg value="git describe --abbrev=0 | tr -d v" />
-		</exec>
-            <echo>Full Version: ${fullVersion}</echo>
-            <echo>Version: ${releaseVersion}</echo>
-	</target>
+        <!-- Full version examples: 1.0.0-SNAPSHOT-0-HASH, 1.0.0-0-HASH, 1.0.0-1-HASH, etc -->
+        <exec outputproperty="fullVersion" dir="." executable="sh">
+            <arg value="-c" />
+            <arg value="git describe --long | tr -d v" />
+        </exec>
+        <!-- Version examples: 1.0.0-SNAPSHOT, 1.0.0, etc -->
+        <exec outputproperty="releaseVersion" dir="." executable="sh">
+            <arg value="-c" />
+            <arg value="git describe --abbrev=0 | tr -d v" />
+        </exec>
+        <echo>Full Version: ${fullVersion}</echo>
+        <echo>Version: ${releaseVersion}</echo>
+    </target>
+
+    <target name="dockerImageBuild" depends="prodVersion,createBuildDir,copyAndUnzipServerUpgradeBundle,copyServerUpgradeBundle,createDockerImage" description="Copy result files to build dir">
```

Comment:
I see duplication of logic and it is better to keep separate xml for docker image automation process.